### ISSUE TITLE
Not working on some sites

### ISSUE THREAD

**Issue by d1ys3nk0** [mgrachev/gastly#1]
I got errors for some sites, for example:

```
Gastly.capture('http://invexpress.ru', '/Users/dlysenko/output.png')

TypeError: exception class/object expected
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/gastly-0.2.2/lib/gastly/screenshot.rb:84:in `handle_exception'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/gastly-0.2.2/lib/gastly/screenshot.rb:47:in `capture'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/gastly-0.2.2/lib/gastly.rb:13:in `capture'
    from (irb):1
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.3/lib/rails/commands/console.rb:110:in `start'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.3/lib/rails/commands/console.rb:9:in `start'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.3/lib/rails/commands/commands_tasks.rb:68:in `console'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.3/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.3/lib/rails/commands.rb:17:in `<top (required)>'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:274:in `require'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:274:in `block in require'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:240:in `load_dependency'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:274:in `require'
    from /Users/dlysenko/Projects/magadan/bin/rails:8:in `<top (required)>'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:268:in `load'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:268:in `block in load'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:240:in `load_dependency'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:268:in `load'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from /usr/local/var/rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from -e:1:in `<main>'irb(main):002:0> 
```

Can you help to fix it?

**Comment by mgrachev** [comment]
Hi. The problem was in the JavaScript error on the website invexpress.ru.

I fixed it in the version [0.2.3](https://github.com/mgrachev/gastly/releases/tag/v0.2.3). Please update gem.

Thanks for the help!

**Comment by d1ys3nk0** [comment]
Thanks for this great product.
